pkg/provisionerapi/api.go

```diff
@@ -0,0 +1,99 @@
+package provisionerapi
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/iver-wharf/wharf-cmd/pkg/provisioner"
+	"github.com/iver-wharf/wharf-core/pkg/ginutil"
+	"github.com/iver-wharf/wharf-core/pkg/logger"
+	"github.com/iver-wharf/wharf-core/pkg/logger/consolepretty"
+	"k8s.io/client-go/rest"
+)
+
+var log = logger.NewScoped("PROVISIONER-API")
+var kubeconfig *rest.Config
+
+func Serve(cfg *rest.Config) error {
+	kubeconfig = cfg
+	logger.AddOutput(logger.LevelDebug, consolepretty.Default)
+
+	gin.DefaultWriter = ginutil.DefaultLoggerWriter
+	gin.DefaultErrorWriter = ginutil.DefaultLoggerWriter
+
+	r := gin.New()
+	r.Use(
+		ginutil.DefaultLoggerHandler,
+		ginutil.RecoverProblem,
+	)
+
+	g := r.Group("/api")
+	g.GET("/", func(c *gin.Context) { c.JSON(200, gin.H{"message": "pong"}) })
+	g.GET("/worker", listWorkersHandler)
+	g.POST("/worker", createWorkerHandler)
+	g.DELETE("/worker/:workerId", deleteWorkerHandler)
+
+	bindAddress := "0.0.0.0:8080"
```

Comment:
nit: This can be changed to const

```suggestion
	const bindAddress = "0.0.0.0:8080"
```